pilot/pkg/serviceregistry/consul/conversion.go

```diff
@@ -23,6 +23,8 @@ import (
 
 	"istio.io/istio/pilot/pkg/model"
 	"istio.io/istio/pkg/log"
+
+        meshconfig "istio.io/api/mesh/v1alpha1"
```

Comment:
file needs to have `goimports` run

Reply:
not sure what this means. maybe because I'm very new to golang.